### ISSUE TITLE
Icon Request: fa-instagram-square

### ISSUE THREAD

**Issue by vaninanton** [FortAwesome/Font-Awesome#9223]
It would be great

**Comment by cramae** [comment]
+1

**Comment by andreabrunato** [comment]
Really needed, actually all social icons have it and instagram actual icon (in v. 4.6.3) is outlined, it differs too much from others

**Comment by drewsbrew** [comment]
+1 on the new logo update

**Comment by kincaider** [comment]
+1

**Comment by feriancek** [comment]
+1

**Comment by gmmarc** [comment]
+1

**Comment by kimar** [comment]
+1

**Comment by dangnelson** [comment]
[instagram-square.svg.zip](https://github.com/FortAwesome/Font-Awesome/files/471044/instagram-square.svg.zip)

**Comment by ErykJaroszewicz** [comment]
very needed!

**Comment by fracsi** [comment]
Any news on this?

**Comment by abstractpenguin** [comment]
+1 Would be great for scenarios like this: 
<img width="357" alt="screen shot 2016-11-28 at 11 32 51 am" src="https://cloud.githubusercontent.com/assets/884708/20676668/827ebc96-b55e-11e6-96aa-d7334a31be50.png">

**Comment by Freeburn113** [comment]
+1

**Comment by a-mawlawi** [comment]
+1

**Comment by abhi191** [comment]
+1

**Comment by micodel** [comment]
+1

**Comment by FabriDamazio** [comment]
+1

**Comment by mshahov** [comment]
+1

**Comment by enricopalazzo** [comment]
+2

**Comment by anujmv** [comment]
+1

**Comment by wja123** [comment]
+1

**Comment by gl03** [comment]
+1

**Comment by hsleewis** [comment]
+1

**Comment by zogorn** [comment]
very needed

**Comment by noobunity** [comment]
+1

**Comment by maxkaplan** [comment]
+1

I agree! Very needed - current icon without square is off brand if you are using others that are square as show in @abstractpenguin screenshot.

Thanks!

**Comment by Myth2014** [comment]
+1

As @abstractpenguin  showed in the screenshot. It would be awesome if all social icons could have a -square version.

Thank you!

**Comment by DerJP** [comment]
+1

**Comment by richardbushell** [comment]
+1

**Comment by daniel-souza** [comment]
+1

**Comment by andrea-86** [comment]
+1

**Comment by qasko** [comment]
+1

**Comment by JibbyJames** [comment]
Using SVGs and Masking can be used as a workaround for now:

`<i class="fab fa-instagram" data-fa-transform="shrink-3" data-fa-mask="fas fa-square"></i>`

More info here: https://fontawesome.com/how-to-use/svg-with-js

**Comment by andrea-86** [comment]
But SVGs it's an implementatio of FA5, but in my project i use FA4.
The change of teh sintax represent a biggest work for doing this change in
the site just now.


Andrea Nenci

2018-01-10 16:06 GMT+01:00 James Buckley <notifications@github.com>:

**Comment by brnrd87** [comment]
+1

**Comment by hardikamal** [comment]
+1

**Comment by atwright147** [comment]
👍

**Comment by cyoung502** [comment]
It's 2018 and we still out here hacking the icon into a square with css

**Comment by 0xACB** [comment]
+1

**Comment by irwinbraganza** [comment]
+1

**Comment by zoogits** [comment]
+1

**Comment by brandonmcconnell** [comment]
+1

**Comment by stefenphelps** [comment]
+1

**Comment by cyoung502** [comment]
[From 5.2 announcement:](https://blog.fontawesome.com/font-awesome-5-2-372-new-icons-adfb42c6c35)

> Brands
> We’ve also added the top 6 most requested icons from the icon leaderboard.

Bump for next update.  You guys skipped this in 5.2 and the icon is literally already done, https://github.com/FortAwesome/Font-Awesome/issues/13053.

This issue currently sits at #18 on that list of brands so let's all try and get this to the top, even though Instagram is arguably one of the largest brands and should take precedence.

**Comment by architchandra** [comment]
This would be really helpful. +1 for fa-instagram-square

**Comment by chrisgresh** [comment]
+2

**Comment by YatheeshRaju** [comment]
its 2019 and+1

**Comment by janvitos** [comment]
I saw this post and decided to create an SVG instagram-square icon that I imported in icomoon. Here's the link: https://drive.google.com/open?id=19iUYxrivxs3DPuvF8EvudOcV2MahsoWy

Hope this helps anyone who needs it.

**Comment by drrcknlsn** [comment]
+1

**Comment by jprobert** [comment]
+1

**Comment by bmoredrew** [comment]
+1 on this.

**Comment by carogaspari** [comment]
+1 please!

**Comment by LeaTark** [comment]
+1 Really surprised this one was omitted.  Thought I was doing something wrong.

**Comment by flyingkitlets** [comment]
> +1
> 
> Something that may help :
> [instagram-square.eps.zip](https://github.com/FortAwesome/Font-Awesome/files/274888/instagram-square.eps.zip)

Thanks so much - really does the job!

**Comment by mplungjan** [comment]
+1

**Comment by Jbry123** [comment]
im so dead that this still isnt a thing, COME ON FONT AWESOME

**Comment by Pfeifakopf** [comment]
+1

**Comment by cirilosouza** [comment]
+1

**Comment by fysheets** [comment]
@sensibleworld out of curiosity is there a potential timeline for this item?

**Comment by sensibleworld** [comment]
@fysheets Just waiting on a few final items. Should be out in 5.12.1

**Comment by fysheets** [comment]
Awesome, thank you @sensibleworld!

**Comment by sensibleworld** [comment]
https://fontawesome.com/icons/instagram-square?style=brands

**Comment by gl03** [comment]
haha, congrats! 🎂
